packages/axios-extension/src/abap/types/adt-types.ts

```diff
@@ -57,3 +57,15 @@ export interface AdtCategory {
     term: AdtCategoryTerm;
     scheme: string;
 }
+
+export interface TransportRequest {
+    transportNumber: string;
+    user: string;
+    date: string;
+    time: string;
```

Comment:
If we're using string date and times, what format are the values going to be in?

Reply:
I don't know if backend provides a single consistent date/time type. I try to provide a best effort converter to convert transport request properties in ADT XML response to a typed return value. If this property would be used by anyone, they would need to handle it.

Reply:
Removed properties that we are uncertain about their data format.

Reply:
thx